node/pkg/node/node_test.go

```diff
@@ -263,20 +404,27 @@ func makeObsDb(tc []testCase) mock.ObservationDb {
 
 // #nosec G107 -- it's OK to make http requests with `statusAddr` because `statusAddr` is trusted.
 func testStatusServer(ctx context.Context, logger *zap.Logger, statusAddr string) error {
```

Comment:
Style suggestions for future work like this (not worth blocking this PR whatsoever):
* For functions in tests that have zero assertions, they should not have the name `test` in them
* For functions in test files that aren't tests, please pass `*testing.T` into the function and call [t.Helper()](https://pkg.go.dev/testing#T.Helper) as the first line. This makes stack traces when the tests fails for developers or in CI massively more useful for developers.